recipe/meta.yaml

```diff
@@ -6,20 +6,21 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 5731e09d5f82c7f8e3849b12ec9c7dd0b6afbab1a6c70842e99e5d4478954edb
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
```

Comment:
Given that the `_0` build had failed on `main` (and thus no packages were published), this is a case where you didn't need to bump the build number. It doesn't hurt though, just FYI